upgrading/automated_upgrades.adoc

```diff
@@ -756,6 +797,34 @@ You can set the `openshift_logging_elasticsearch_replace_configmap` parameter to
 ConfigMap can cause the upgrade to fail. The default is set to `false`. For more information, see the parameter 
 in xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[specify logging Ansible variables].
 
+. Scale down your Fluentd and Elasticsearch pods to stop data ingestion and ensure the cluster state does not change.
+
+. Perform an link:https://www.elastic.co/guide/en/elasticsearch/reference/2.4/indices-flush.html[Elasticsearch Index flush] on all relevant indices. The flush process persists all logs from memory to disk, which prevents log loss when Elasticsearch is shutdown during the upgrade.
+
+. Use the link:https://www.elastic.co/guide/en/elasticsearch/reference/5.5/modules-snapshots.html[Elasticsearch snapshot module] to create a snapshot of your AWS volumes.
```

Comment:
@mburke5678  'snapshot of your AWS volumes'? Is this document for AWS platform only?

Reply:
@anpingli This was from @jcantrill comment:
_scaling down fluent will stop data ingestion but scaling down ES will also ensure the cluster state is not changing so you could say snapshot the AWS volumes_
https://github.com/openshift/openshift-docs/pull/17931#discussion_r348196409

What should the user snapshot? Elasticsearch volumes?

Reply:
@mburke5678 @jcantrill There are two different snapshots:

First, Elasticsearch has its own [Snapshot&Restore API](https://www.elastic.co/guide/en/elasticsearch/reference/5.5/modules-snapshots.html) and you tell it which indices you want to snapshot to repository (mounted storage). ES has to be running.

Second, there is [AWS snapshot](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSSnapshots.html) and this is completely orthogonal to ES. However, to make sure the snapshot is consistent ES must be stopped. You need to snapshot all the data folders (it holds both the index data and cluster metadata).

I think Jeff was talking about the second scenario, in which case we should remove link to ES Snapshot&Restore API from the doc.

Reply:
@jcantrill ^^

Reply:
We have no experience in Elasticsearch snapshots and are advising user's to back up the actual disk in any way that supports their platform, AWS snapshots is one of them.  I would expect there to be interruption in logging where user's bring down their entire ES cluster, backup disk, migrate

Reply:
"To protect your  Elasticsearch data, you should back up your data using your standard policies" Something like that?

Reply:
What is QE testing, as part of our backup plan in this area? 

@xltian ?

Reply:
@sferich888  That is an offline backup.  Logging QE didn't test the backup/restore in our testing.